Examples/Examples/FormViewExampleView.swift

```diff
@@ -160,6 +166,10 @@ private extension URL {
             return
         }
         
+        validationIsForced = true
+        
+        guard featureForm?.validationErrors.isEmpty ?? true else { return }
```

Comment:
The feature form is already unwrapped on line `161` to get the feature. Instead of unwrapping it a second time, store the unwrapped feature form the first time it is unwrapped:

```swift
func submitChanges() async {
    guard let featureForm = featureForm,
          let table = featureForm.feature.table as? ServiceFeatureTable,
          table.isEditable,
          let database = table.serviceGeodatabase else {
        print("A precondition to submit the changes wasn't met.")
        return
    }
    
    ...
    
    self.featureForm = nil
}
```